boards/native/nrf_bsim/board.yml

```diff
@@ -0,0 +1,13 @@
+boards:
+- name: nrf52_bsim
+  vendor: Zephyr
+  socs:
+  - name: native
+- name: nrf5340bsim_nrf5340_cpuapp
```

Comment:
this ought to be a soc with a cpucluster if the intention is to have it appear similar to a nRF530 SoC.

Doing so would also remove some other comments which I would otherwise have regarding the Kconfig.

I did notice: 
> Note: the nrf5340 variants remain as their own
> targets, instead of being variants of the base ones
> to avoid breakage in this commit

what kind of breakage are you refering to, and are you planning to make more updates to the bsim boards with regards to variants ?

Reply:
Same answer as above
(clarification w "variants" I meant "socs" or "cpu-clusters")